catalog/plugin/catalog-plugin-metacard-validation/src/main/java/ddf/catalog/metacard/validation/MetacardValidityMarkerPlugin.java

```diff
@@ -159,20 +159,18 @@ public UpdateRequest process(UpdateRequest input)
 
     metacard.setAttribute(
         new AttributeImpl(
-            Validation.VALIDATION_ERRORS,
-            (List<Serializable>) new ArrayList<Serializable>(newErrors)));
+            Validation.VALIDATION_ERRORS, (List<Serializable>) new ArrayList<>(newErrors)));
```

Comment:
✏️ You could use ApacheCommons' `SetUniqueList` decorator to avoid casting and make our intent a little more explicit. Pass the original serializable lists here and call `asSet()` for the validator construct.

Really though it's just unfortunate that our interfaces conflict this way.